runtime/doc/treesitter.txt

```diff
@@ -979,6 +982,11 @@ get_filetypes({lang})                *vim.treesitter.language.get_filetypes()*
         (`string[]`) filetypes
 
 get_lang({filetype})                      *vim.treesitter.language.get_lang()*
+    Returns the language name to be used when loading a parser for {filetype}.
```

Comment:
Future note: This appears to be the dual of `register()`. Can we name register() to `set_lang()` ? 

We do not use the name "register" anywhere else in stdlib AFAICT, and this thankfully looks like the last such usage.

Reply:
No, `set_lang` implies you set _the_ language; this adds it to a "register". I don't thinks there's a precedent in the API for this.

Reply:
we don't have `add_lang`.

Reply:
True, and that could work, but I feel that that would be confusing with `vim.treesitter.language.add` (which is... not a great name).

To be blunt: without a complete rethink of the API, this sort of thing is putting lipstick on a pig...